_clients/data-prepper/configure-logstash-data-prepper.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Configure Logstash for Data Prepper
+parent: Data Prepper
+nav_order: 2
+---
+# Configure Logstash for Data Prepper
+You can run Data Prepper with a Logstash configuration.
+
+As mentioned in the [Getting Started]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/getting-started) guide, you'll need to configure Data Prepper with a pipeline using a `pipelines.yaml` file.
+
+Alternatively, if you have a Logstash configuration `logstash.conf` to configure Data Prepper instead of `pipelines.yaml`.
+
+## Supported plugins
+
+As of the Data Prepper 1.2 release, the following plugins from the Logstash configuration are supported:
+* HTTP Input plugin
+* Grok Filter plugin
+* Elasticsearch Output plugin
+* Amazon Elasticsearch Output plugin
+
+## Limitations
+* Apart from the supported plugins, all other plugins from the Logstash configuration will throw an `Exception` and fail to run.
+* Conditionals in the Logstash configuration are not supported as of the Data Prepper 1.2 release.
+
+## Running Data Prepper with a Logstash configuration
+
+1. To install Data Prepper's Docker image, see the _Installation_ section in the [Getting Started]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/getting_started) guide.
```

Comment:
I think we need a command here, like "Install the Data Prepper Docker image. For instructions, see .... 

Remove "guide".